packages/vest/src/typings/vest.d.ts

```diff
@@ -258,6 +259,7 @@ type TEnforceLazy = {
     [key: string]: LazyEnforceValue | LazyEnforceValue[];
   }) => LazyEnforceValue;
   optional: <T>(...rules: LazyEnforceValue[]) => LazyEnforceValue;
+  anyOf: <T>(...rules: LazyEnforceValue[]) => LazyEnforceValue;
```

Comment:
You're also going to get a merge conflict here. I renamed this key.
```suggestion
anyOf: <T>(...rules: TEnforceLazy[]) => TEnforceLazy;
```